demos/link-prediction/random-walks/ctdne-link-prediction.ipynb

```diff
@@ -0,0 +1,680 @@
+{
```

Comment:
<p>I think it should be mentioned here that CTDNE method can potentially miss certain nodes in the walks and hence there won't be any embeddings calculated for them. This issue is being  fixed here by assigning embeddings vectors of 0's for the nodes not visited in any of the walks. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1056/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>

Reply:
Thanks, I've added a bit of an explanation